### ISSUE TITLE
Privacy - Translation requests tracked? 

### ISSUE THREAD

**Issue by mehmedes** [ModernMT/MMT#199]
Is it by any means possible to trace back or keep track of translation requests. 

Suppose someone got into my server, could he find any coherent sentences after I deleted the test and dev corpora.

**Comment by davidecaroselli** [comment]
Hi @mehmedes 

by default MMT tracks down translation requests in the log file `runtime/default/logs/node.log`. If you want to prevent the system to log any info, you can run MMT with a different verbosity:
```
./mmt start -v 0
```

**Comment by marcotrombetti** [comment]
@davidecaroselli @mehmedes 

When you use MMT evaluate we are sending the test set to Google Translate for performance comparison. So if confidential data is on your training data, avoid using MMT Evaluate vs GT.
